src/main/java/org/bookstore/exceptions/GlobalExceptionHandler.java

```diff
@@ -35,6 +35,22 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(
         return new ResponseEntity<>(body, headers, status);
     }
 
+    @ExceptionHandler(value = {EntityNotFoundException.class})
+    private ResponseEntity<Object> handleEntityNotFound(EntityNotFoundException ex) {
+        Map<String, Object> body = new LinkedHashMap<>();
+        body.put("errors", ex.getMessage());
+        HttpStatusCode status = HttpStatus.NOT_FOUND;
+        return new ResponseEntity<>(body, status);
+    }
+
+    @ExceptionHandler(value = {DataProcessingException.class})
+    private ResponseEntity<Object> handleDataProcessingException(EntityNotFoundException ex) {
```

Comment:
looks weird

handler claims to handle `DataProcessingException`, although parameter is `EntityNotFoundException`


P.S. don't make handlers as private methods